resources/deu.ini

```diff
@@ -1,208 +1,219 @@
 
 ; The /p denyplayer Subcommand
 denyplayer.name=denyplayer
 denyplayer.alias=deny
-denyplayer.desc=Verbiete einem Spieler deinen Plot zu betreten
-denyplayer.usage=/p deny <player>
+denyplayer.desc=Verbiete einem Spieler dein Grundstück zu betreten.
+denyplayer.usage=/p deny <Spieler>
 denyplayer.helper={%0} ist ein Helfer!
-denyplayer.alreadyone={%0} ist es bereits verboten deinen Plot zu betreten!
-denyplayer.notaplayer=Dieser Spieler ist gerade nicht online!
-denyplayer.cannotdeny=Du kannst {%0} nicht verbieten deinen Plot zu betreten!
-denyplayer.attempteddeny={%0} versucht dich von deinem Plot zu sperren!
-denyplayer.success1={%0} darf ab jetzt deinen Plot nicht mehr betreten!
-denyplayer.success2=Du darfst ab jetzt den Plot  {%0};{%1} von {%2} nicht mehr betreten!
+denyplayer.alreadyone={%0} ist es bereits verboten, dein Grundstück zu betreten!
+denyplayer.notaplayer=Dieser Spieler ist momentan nicht online.
+denyplayer.cannotdeny=Du kannst {%0} nicht verbieten, dein Grundstück zu betreten!
+denyplayer.attempteddeny={%0} versucht, dich von deinem Grundstück zu sperren!
+denyplayer.success1={%0} darf dein Grundstück nun nicht mehr betreten.
+denyplayer.success2=Du darfst nicht mehr das Grundstück {%0};{%1} von {%2} betreten!
+
 ; The /p undenyplayer Subcommand
 undenyplayer.name=undenyplayer
 undenyplayer.alias=undeny
-undenyplayer.desc=Entsperrt einen Spieler von deinem Plot
+undenyplayer.desc=Entsperre einen Spieler von deinem Grundstück.
 undenyplayer.usage=/p undeny <player>
 undenyplayer.notone={%0} ist bereits entsperrt!
-undenyplayer.notaplayer=Dieser Spieler ist gerade nicht online!
+undenyplayer.notaplayer=Dieser Spieler ist momentan nicht online!
 undenyplayer.success1={%0} ist nun entsperrt!
-undenyplayer.success2=Du darfst den Plot {%0};{%1} von {%2} jetzt wieder betreten!
+undenyplayer.success2=Du darfst das Grundstück {%0};{%1} von {%2} nun wieder betreten!
 
 ; The /p claim Subcommand
 claim.name=claim
-claim.desc=Beansprucht den Plot auf dem du stehst
-claim.usage=/p claim [name]
-claim.yourplot=Dir gehört dieser Plot bereits!
-claim.alreadyclaimed=Dieser Plot gehört bereits {%0}
-claim.maxplots=Du hast das Limit von {%0} Plots pro Spieler überschritten!
-claim.nomoney=Du hast nicht genug Geld um diesen Plot zu kaufen!
-claim.success=Dir gehört dieser Plot jetzt!
+claim.desc=Beanspruche das Grundstück, auf dem du dich befindest.
+claim.usage=/p claim [Name]
+claim.yourplot=Dir gehört dieses Grundstück bereits!
+claim.alreadyclaimed=Dieses Grundstück gehört bereits {%0}.
+claim.maxplots=Du darfst das Limit von {%0} Grundstücken pro Spieler nicht überschreiten!
+claim.nomoney=Du besitzt nicht genug Geld, um dieses Grundstück zu kaufen!
+claim.success=Du hast dieses Grundstück erfolgreich für dich beansprucht.
+
 ; The /p auto Subcommand
 auto.name=auto
 auto.alias=a
-auto.desc=Teleportiert dich zu den nächsten verfügbaren Plot
+auto.desc=Teleportiere dich zu dem nächsten, freien Grundstück.
```

Comment:
Since it's for a usage, I would prefer the shorter word "zum" instead of "zu dem"